packages/e2e/local-network/scripts/mint-handles.sh

```diff
@@ -70,3 +70,63 @@ cardano-cli transaction sign \
   --out-file handle-tx.signed
 
 cardano-cli transaction submit --testnet-magic 888 --tx-file handle-tx.signed
+
+sleep 10
```

Comment:
Why is this needed and why 10? :sweat_smile:

Reply:
it will take outdated utxo othervise

Reply:
Sounds like we could see some flakiness due to this. Can we implement loop with sleep 1 that checks for state change instead?